adafruit_displayio_ssd1306.py

```diff
@@ -73,15 +75,24 @@ def __init__(
         # Patch the init sequence for 32 pixel high displays.
         init_sequence = bytearray(_INIT_SEQUENCE)
         height = kwargs["height"]
+        width = kwargs["width"]
         if "rotation" in kwargs and kwargs["rotation"] % 180 != 0:
             height = kwargs["width"]
+            width = kwargs["height"]
         init_sequence[16] = height - 1  # patch mux ratio
-        if kwargs["height"] == 32:
+        if height == 32 and width == 64:  # FIX ME
```

Comment:
What is this FIX ME note for?

Reply:
Code I have found for CircuitPython but without DisplayIO or for Arduino do not seems to need and exception for 64x32.
So this is odd to have that special value there ...
There is possibly something wrong, or my 64x32 is a different kind. I hope to get feedback from owner of other such display, and other size of display too.